indra/llcharacter/lljoint.cpp

```diff
@@ -258,17 +258,16 @@ LLJoint *LLJoint::findJoint( const std::string &name )
 	if (name == getName())
 		return this;
 
-	for (joints_t::const_iterator iter = mChildren.begin();
-		 iter != mChildren.end(); ++iter)
+	for (LLJoint* joint : mChildren)
```

Comment:
Prefer the `for (item : contianer)` pattern.  It tends to be more readable and terse.  When `auto` is used for `item`'s **typename** it can even be container agnostic.